### PR TITLE
Added CacheResolver to underlying ProxyResolver

### PR DESCRIPTION
**Pull request by denmit**

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -71,6 +71,7 @@ public function getConfigTreeBuilder()
                                 ->arrayNode('proxies')
                                     ->prototype('scalar')->end()
                                 ->end()
+                                ->scalarNode('cache')->defaultValue('null')->end()
```
Comment (rvanlaarhoven) [inline DependencyInjection/Configuration.php]: Can you explain what this option is for and where it is used?

Comment (denmit) [inline DependencyInjection/Configuration.php]: http://symfony.com/doc/current/bundles/LiipImagineBundle/cache-resolver/aws_s3.html#additional-options

According to documentation it's for enabling cache itself, but ommiting it or setting it to false will still cache data, so will remove this one.

File: Resources/config/services.yml [hunk 2]
```diff
@@ -94,3 +94,16 @@ services:
             - %opifer_media.storages.aws_s3.proxies%
         tags:
             - { name: 'liip_imagine.cache.resolver', resolver: 'proxy_aws_storage' }
+    
+    cache.apccache:
```
Comment (rvanlaarhoven) [inline Resources/config/services.yml]: Could you change this naming to `opifer.cache.apc`?

Comment (denmit) [inline Resources/config/services.yml]: Renamed apccache service.

**Comment by rvanlaarhoven** [comment]
Thanks!

**Pull request merged**